contrib/automation_tests/orbit_stadia_symbol_store.py

```diff
@@ -0,0 +1,57 @@
+"""
+Copyright (c) 2022 The Orbit Authors. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+"""
+
+import os
+
+from absl import app
+
+from core.orbit_e2e import E2ETestSuite
+from test_cases.connection_window import FilterAndSelectFirstProcess, ConnectToStadiaInstance
+from test_cases.symbols_tab import ClearSymbolCache, LoadSymbols, WaitForLoadingSymbolsAndCheckModule
+from test_cases.symbol_locations import ClearAllSymbolLocations, EnableStadiaSymbolStore
+from test_cases.main_window import EndSession
+"""
+Test symbol loading from Stadia symbol store.
+
+Before this script is run there needs to be a gamelet reserved and
+"hello_ggp_standalone" has to be started. 
+Also Orbit needs to be started with the command line argument "--disable_instance_symbols".
+
+The script requires absl and pywinauto. Since pywinauto requires the bitness of
+the python installation to match the bitness of the program under test, it needs
+to be run from 64 bit python.
+
+This automation script covers a basic workflow:
+ - (pre clean up) connect to the gamelet and start session, 
+   disable stadia symbol store, clear the symbol cache and delete all symbol locations
+ - restart session
+ - select the process "hello_ggp_standalone"
+ - verify symbol loading fails
+ - enable stadia symbol store
+ - verify symbol loading works
+ - (post clean up) disable stadia symbol store
+"""
+
+def main(argv):
+    test_cases = [
+        ConnectToStadiaInstance(),
+        FilterAndSelectFirstProcess(process_filter='hello_ggp'),
+        EnableStadiaSymbolStore(enable_stadia_symbol_store=False),
+        ClearAllSymbolLocations(),
+        ClearSymbolCache(),
+        EndSession(),
+        FilterAndSelectFirstProcess(process_filter='hello_ggp'),
+        LoadSymbols(module_search_string="libggp", expect_fail=True),
+        EnableStadiaSymbolStore(enable_stadia_symbol_store=True),
+        WaitForLoadingSymbolsAndCheckModule(module_search_string="libggp"),
+        EnableStadiaSymbolStore(enable_stadia_symbol_store=False)
```

Comment:
As talked offline, you could add tests for loaded capture in between this. Something like this should work:
```
ClearSymbolCache()
EndSession()
LoadLatestCapture()
WaitForLoadingSymbolsAndCheckModule(module_search_string="libggp")
```

Reply:
If we load capture from file, nothing showing in Modules and Functions view, so we are not able to verify that symbols are loaded